src/platforms/javascript/common/install/cdn.mdx

```diff
@@ -1,11 +1,6 @@
 ---
-title: CDN Installation
-sidebar_order: 1
-caseStyle: camelCase
-supportLevel: production
-categories:
-  - browser
-description: "Learn more about installation and configuration for those using a content delivery network (CDN)."
```

Comment:
Do we need to keep this description for the automated page?

Reply:
nope no descriptions are _needed_ they're just an optional thing. I only removed it because the other do didn't have thorough descriptions (npm has none, lazy loader was inaccurate)